geopyspark/geotrellis/layer.py

```diff
@@ -1010,9 +1075,80 @@ def __init__(self, layer_type, srdd):
         self.layer_metadata = Metadata.from_dict(json.loads(self.srdd.layerMetadata()))
         self.zoom_level = self.srdd.getZoom()
 
+    @classmethod
+    def read_to_layout(cls,
```

Comment:
Is `read_to_layout` final choice for the name? I recall it was said that existence is layout is implied by the class name in `TiledRasterLayer.read_to_layout` but not sure if that argument for calling out in difference made a bigger sway.

Reply:
That wasn't a final choice, just something we initially thought of. I agree that the `layout` portion is implicitly understood, and I have no issue changing the name of the method to `read`.

Reply:
I don't have a hard preference in mind. What do you think is best for the API ?

Reply:
I think making it `read` would be good. Just so that the API is more consistent.